eras/shelley/impl/src/Cardano/Ledger/Shelley/TestUtils.hs

```diff
@@ -0,0 +1,293 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Cardano.Ledger.Shelley.TestUtils
```

Comment:
We should put this new module in a test package, and prefix the namespace with `Test`.

Reply:
The test module is not in a module that can be imported. So we put it here.

Reply:
I only see this module being used in the packages `cardano-ledger-test` and `cardano-ledger-shelley-test`. I don't see what prevents us from placing this module in the library in `cardano-ledger-shelley-test`. am I missing something?

Reply:
how about:

`eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Rewards.hs`

Reply:
moved to  Test.Cardano.Ledger.Shelley.Rewards